bootstrap.py

```diff
@@ -1053,12 +1066,78 @@ def exec_service(service, command, failonerror=True):
         migrate_systems(options.org, options.activationkey)
         if options.enablerepos:
             enable_repos()
+    elif options.new_capsule and options.foreman_fqdn:
+        # > ELIF new_capsule and foreman_fqdn set, will migrate to other capsule
+        #
+        # > will replace CA certificate, reinstall katello-agent, gofer
+        # > will optionally update hostgroup and location
+        # > wil update system definition to point to new capsule for content,
+        # > Puppet, OpenSCAP and update Puppet configuration (if applicable)
+        # > MANUAL SIGNING OF CSR OR MANUALLY CREATING AUTO-SIGN RULE STILL REQUIRED!
+        # > API doesn't have a public endpoint for creating auto-sign entries yet!
+
+        # Make system ready for switch, gather required data
+        get_bootstrap_rpm(clean=True, unreg=False)
+        install_katello_agent()
+        API_PORT = get_api_port()
+        capsule_id = return_matching_foreman_key('smart_proxies', 'name="%s"' % options.foreman_fqdn, 'id', False)
+        host_id = return_matching_foreman_key('hosts', 'name="%s"' % FQDN, 'id', False)
+        capsule_features = [f['name'] for f in get_json("https://" + options.foreman_fqdn + ":" + API_PORT + "/katello/api/capsules/%s" % capsule_id)['features']]
```

Comment:
can this be an own function please?

Reply:
I'm assuming you specifically mean the  capsule_features part? No sense in moving the whole part into a function imho.

Reply:
yeah, I meant the get features part :)